<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="2" total="18" passed="14">
  <reporter-output>
    <line>
      <![CDATA[Test Method resides in com.Sample.NewTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.NewTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.TestSample]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.TestSample]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.TestSample]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.TestSample]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.TestSample]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
    <line>
      <![CDATA[Test Method resides in com.Sample.AppTest]]>
    </line>
  </reporter-output>
  <suite name="AppointmentTests" duration-ms="12653" started-at="2016-12-14T05:48:52Z" finished-at="2016-12-14T05:49:05Z">
    <groups>
    </groups>
    <test name="testGuru3" duration-ms="6852" started-at="2016-12-14T05:48:52Z" finished-at="2016-12-14T05:48:59Z">
      <class name="com.Sample.TestSample">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="setup" is-config="true" duration-ms="3044" started-at="2016-12-14T11:18:52Z" finished-at="2016-12-14T11:18:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="TestSampleTest()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest" duration-ms="1" started-at="2016-12-14T11:18:55Z" finished-at="2016-12-14T11:18:55Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.Sample.TestSample.TestSampleTest(TestSample.java:11)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1211)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.TestSample]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest -->
        <test-method status="PASS" signature="TestSampleTest2()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest2" duration-ms="0" started-at="2016-12-14T11:18:56Z" finished-at="2016-12-14T11:18:56Z">
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.TestSample]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest2 -->
        <test-method status="PASS" signature="TestSampleTest3()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest3" duration-ms="1" started-at="2016-12-14T11:18:57Z" finished-at="2016-12-14T11:18:57Z">
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.TestSample]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest3 -->
        <test-method status="FAIL" signature="TestSampleTest4()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest4" duration-ms="1" started-at="2016-12-14T11:18:57Z" finished-at="2016-12-14T11:18:57Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.Sample.TestSample.TestSampleTest4(TestSample.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1211)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.TestSample]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest4 -->
        <test-method status="PASS" signature="TestSampleTest6()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest6" duration-ms="1" started-at="2016-12-14T11:18:58Z" finished-at="2016-12-14T11:18:58Z">
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.TestSample]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest6 -->
        <test-method status="SKIP" signature="TestSampleTest1()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest1" duration-ms="0" started-at="2016-12-14T11:18:58Z" depends-on-methods="com.Sample.TestSample.TestSampleTest" finished-at="2016-12-14T11:18:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestSample.TestSampleTest1()[pri:0, instance:com.Sample.TestSample@1ee9989] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestSample.TestSampleTest1()[pri:0, instance:com.Sample.TestSample@1ee9989] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1211)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestSampleTest1 -->
        <test-method status="SKIP" signature="TestSampleTest5()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="TestSampleTest5" duration-ms="0" started-at="2016-12-14T11:18:58Z" depends-on-methods="com.Sample.TestSample.TestSampleTest" finished-at="2016-12-14T11:18:58Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestSample.TestSampleTest5()[pri:0, instance:com.Sample.TestSample@1ee9989] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestSample.TestSampleTest5()[pri:0, instance:com.Sample.TestSample@1ee9989] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1033)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1211)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestSampleTest5 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.Sample.TestSample@1ee9989]" name="tearDown" is-config="true" duration-ms="436" started-at="2016-12-14T11:18:59Z" finished-at="2016-12-14T11:18:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Sample.TestSample -->
      <class name="com.Sample.AppTest">
        <test-method status="PASS" signature="TestPass()[pri:0, instance:com.Sample.AppTest@13ad2d6]" name="TestPass" duration-ms="1" started-at="2016-12-14T11:18:58Z" finished-at="2016-12-14T11:18:58Z">
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestPass -->
      </class> <!-- com.Sample.AppTest -->
    </test> <!-- testGuru3 -->
    <test name="testGuru5" duration-ms="5730" started-at="2016-12-14T05:48:59Z" finished-at="2016-12-14T05:49:05Z">
      <class name="com.Sample.AppTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.Sample.AppTest@1a5c39e]" name="setup" is-config="true" duration-ms="3174" started-at="2016-12-14T11:18:59Z" finished-at="2016-12-14T11:19:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@1a5c39e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:02Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass1]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@1a5c39e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:03Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass2]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@1a5c39e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:04Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass3]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@1a5c39e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:04Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass4]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.Sample.AppTest@1a5c39e]" name="tearDown" is-config="true" duration-ms="376" started-at="2016-12-14T11:19:05Z" finished-at="2016-12-14T11:19:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Sample.AppTest -->
    </test> <!-- testGuru5 -->
  </suite> <!-- AppointmentTests -->
  <suite name="ContactTests" duration-ms="20116" started-at="2016-12-14T05:48:32Z" finished-at="2016-12-14T05:48:52Z">
    <groups>
    </groups>
    <test name="testGuru" duration-ms="20116" started-at="2016-12-14T05:48:32Z" finished-at="2016-12-14T05:48:52Z">
      <class name="com.Sample.NewTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.Sample.NewTest@3f4d3d]" name="setup" is-config="true" duration-ms="16844" started-at="2016-12-14T11:18:32Z" finished-at="2016-12-14T11:18:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="TestSampleTest()[pri:0, instance:com.Sample.NewTest@3f4d3d]" name="TestSampleTest" duration-ms="7" started-at="2016-12-14T11:18:49Z" finished-at="2016-12-14T11:18:49Z">
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.NewTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest -->
        <test-method status="PASS" signature="TestSampleTest1()[pri:0, instance:com.Sample.NewTest@3f4d3d]" name="TestSampleTest1" duration-ms="2" started-at="2016-12-14T11:18:50Z" finished-at="2016-12-14T11:18:50Z">
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.NewTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- TestSampleTest1 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.Sample.NewTest@3f4d3d]" name="tearDown" is-config="true" duration-ms="1915" started-at="2016-12-14T11:18:50Z" finished-at="2016-12-14T11:18:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Sample.NewTest -->
    </test> <!-- testGuru -->
  </suite> <!-- ContactTests -->
  <suite name="QASourceRegressionSuite" duration-ms="0" started-at="2016-12-14T05:49:11Z" finished-at="2016-12-14T05:49:11Z">
    <groups>
    </groups>
  </suite> <!-- QASourceRegressionSuite -->
  <suite name="MeetingTests" duration-ms="5505" started-at="2016-12-14T05:49:05Z" finished-at="2016-12-14T05:49:11Z">
    <groups>
    </groups>
    <test name="testGuru1" duration-ms="5505" started-at="2016-12-14T05:49:05Z" finished-at="2016-12-14T05:49:11Z">
      <class name="com.Sample.AppTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.Sample.AppTest@6451e]" name="setup" is-config="true" duration-ms="2992" started-at="2016-12-14T11:19:05Z" finished-at="2016-12-14T11:19:08Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@6451e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:08Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass1]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@6451e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:09Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass2]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@6451e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:09Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:09Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass3]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="LogIn_Test(java.lang.String, java.lang.String)[pri:0, instance:com.Sample.AppTest@6451e]" name="LogIn_Test" duration-ms="1" started-at="2016-12-14T11:19:10Z" data-provider="LoginCredentials" finished-at="2016-12-14T11:19:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserId4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Pass4]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Test Method resides in com.Sample.AppTest]]>
            </line>
          </reporter-output>
        </test-method> <!-- LogIn_Test -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.Sample.AppTest@6451e]" name="tearDown" is-config="true" duration-ms="392" started-at="2016-12-14T11:19:10Z" finished-at="2016-12-14T11:19:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.Sample.AppTest -->
    </test> <!-- testGuru1 -->
  </suite> <!-- MeetingTests -->
</testng-results>
